### ISSUE TITLE
rules aliases

### ISSUE THREAD

**Issue by tal-rofe** [Vinyl-Depository/Inflint#9]
### Description

Same idea as normal aliases for convention, but also for file patterns.
Applies to folders as well.

This way can configure:
`{ ruleAliases: { "*.{png,jpeg,jpg}": "[img]" }, rules: { "[img]": 2 } }`
This will forbid any image, for example. 

### Proposed solution

add rule aliases feature

### Do you want to create a pull request?

Yes

**Comment by tal-rofe** [comment]
Won't do
useless feature because each alias can be used only once in the rules field..
